### DISCUSSION TITLE
which of the multiple TTFs do you get with GetFont?

### DISCUSSION THREAD

**Discussion by zissakos** [rds1983/FontStashSharp#11] [General]
In your example you add 3 TTFs with fontSystem.AddFont()
Then you only call fontSystem.GetFont(size)
which of the 3 do you get?

**Comment by rds1983** [comment]
All of them.
Multiple ttfs are used when you render a glyph for the first time. As it looks over all of those ttfs until it finds one containing the requested glyph. In the provided example, every ttf provides one type of glyphs(ordinary, Japanese and emojis). If you exclude say Japanese ttf, then it wont be able to render Japanese characters.
See this code for more details: https://github.com/rds1983/FontStashSharp/blob/7ec7dbff73eb0826fb1a830fcb2c5bb671095c08/src/FontStashSharp/FontSystem.cs#L190

**Comment by zissakos** [comment]
so if I want to use different fonts in my game (like Arial, Tahoma, Times etc.) I need multiple FontSystems, right? My notion that you just need one global FontSystem and then switch between fonts is wrong, correct?

**Comment by rds1983** [comment]
yes and yes.
